P5/Source/Specs/path.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright TEI Consortium. 
+Dual-licensed under CC-by and BSD2 licences 
+See the file COPYING.txt for details
+$Date$
+$Id$
+-->
+<?xml-model href="http://jenkins.tei-c.org/job/TEIP5-dev/lastSuccessfulBuild/artifact/P5/release/xml/tei/odd/p5.nvdl" type="application/xml" schematypens="http://purl.oclc.org/dsdl/nvdl/ns/structure/1.0"?>
+<elementSpec xmlns="http://www.tei-c.org/ns/1.0" 
+  xmlns:rng="http://relaxng.org/ns/structure/1.0" ident="path" module="transcr"  
+  xmlns:sch="http://purl.oclc.org/dsdl/schematron">
+  <desc versionDate="2018-03-28" xml:lang="en">defines any line passing through two or more points within a <gi>surface</gi>
+element.</desc>
+  <classes>
+    <memberOf key="att.global"/>
+    <memberOf key="att.coordinated"/>
+    <memberOf key="att.typed"/>
+    <memberOf key="att.written"/>
+    <memberOf key="model.linePart"/>
+  </classes>
+  <content>
+    <empty/>
+  </content>
+  
+<!-- Thanks to Syd B for this approach -->
+  <constraintSpec ident="pathmustnotbeclosed" scheme="schematron">
+    <constraint>
+      <sch:rule context="tei:path[@points]">
+        <!--
+        For a <path> element, we should ensure that the last coordinate does not
+        repeat the first coordinate, otherwise we have a closed polygon, not a
+        path.
+      -->
+        <sch:let name="firstPair" value="tokenize( normalize-space( @points ), '&#x20;')[1]"/>
+        <sch:let name="lastPair"  value="tokenize( normalize-space( @points ), '&#x20;')[last()]"/>
+        <sch:let name="firstX"  value="xs:float( substring-before( $firstPair, ',') )"/>
+        <sch:let name="firstY" value="xs:float( substring-after( $firstPair, ',') )"/>
+        <sch:let name="lastX"   value="xs:float( substring-before( $lastPair, ',') )"/>
+        <sch:let name="lastY"  value="xs:float( substring-after( $lastPair, ',') )"/>
+        <sch:report test="$firstX eq $lastX  and  $firstY eq $lastY">The first and
+          last elements of your path are the same. If you want to specify a closed polygon, use
```

Comment:
Perhaps less important in schematron reports, but I would still prefer something slightly more formal, e.g.: "To specify a closed polygon, use the zone element instead"

Reply:
Again, no problem with the change, but I do tend to think that human-friendly language is more helpful than excessive formality.